SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-or-later

int name, namelen; is wrong for mes libc, it is char* name, so we modify tinycc
to reflect this.

diff --git lib/sh/oslib.c lib/sh/oslib.c
index 90d7be9..37fdf2a 100644
--- lib/sh/oslib.c
+++ lib/sh/oslib.c
@@ -192,8 +192,7 @@ bzero (s, n)
 #    include <sys/utsname.h>
 int
 gethostname (name, namelen)
-     char *name;
-     int namelen;
+     char *name; int namelen;
 {
   int i;
   struct utsname ut;
@@ -209,7 +208,7 @@ gethostname (name, namelen)
 #  else /* !HAVE_UNAME */
 int
 gethostname (name, namelen)
-     int name, namelen;
+     char* name; int namelen;
 {
   strncpy (name, "unknown", namelen);
   name[namelen] = '\0';
